substrate/pkg/controller/substrate/cluster/instance.go

```diff
@@ -78,7 +78,7 @@ func (i *Instance) Create(ctx context.Context, substrate *v1alpha1.Substrate) (r
 		if strings.Contains(aws.StringValue(err.ErrorMessage), "Invalid IAM Instance Profile name") {
 			return reconcile.Result{Requeue: true}, nil
 		}
-		return reconcile.Result{}, fmt.Errorf("creating fleet", aws.StringValue(err.ErrorMessage))
+		return reconcile.Result{}, fmt.Errorf("creating fleet %v", aws.StringValue(err.ErrorMessage))
```

Comment:
Cool -- looks like %v has cleaner output than %s for nil values: https://stackoverflow.com/questions/61283248/format-errors-in-go-s-v-or-w. I should start using this.